src/points/PointsHistoryBottomSheet.test.tsx

```diff
@@ -106,7 +106,7 @@ describe(PointsHistoryBottomSheet, () => {
     expect(tree.getByTestId('PointsHistoryBottomSheet/Empty')).toBeTruthy()
   })
 
-  it('displays content while loading', async () => {
+  it.skip('displays content while loading', async () => {
```

Comment:
`// eslint-disable-next-line jest/no-disabled-tests`

should fix the lint error